### ISSUE TITLE
Add empty alt attribute to generated image

### ISSUE THREAD

**Issue by yatil** [louisremi/background-size-polyfill#4]
I’ve just skimmed the .htc code and it appears that the created img element has no alt attribute. That causes some screen readers to read the filename, which is awkward. Add an empty alt attribute fixes this.

**Comment by louisremi** [comment]
This is a valid concern.
Would you be able and willing to fix this problem?

**Comment by yatil** [comment]
Sent a pull request, see Issue #5.
